I have completed Assignment 3

AmmarMahzan opened the pull request:
## What changes are you trying to make? (e.g. Adding or removing code, refactoring existing code, adding reports)
- Assignment - 3 containing 2 data visualisation.

## What did you learn from the changes you have made?
- Gained experience cleaning timestamp data and extracting temporal features using pandas.
- Learned how to apply design and accessibility principles in real-world data visualizations.
-Practiced communicating data stories clearly through both visual and written explanations.

## Was there another approach you were thinking about making? If so, what approach(es) were you thinking of?
Initially considered using Tableau Public, but I've opted to use Python for both plots within Jupyter Notebook.
## Were there any challenges? If so, what issue(s) did you face? How did you overcome it?
Choosing the suitable dataset
## How were these changes tested?
All code was run in Jupyter Notebook with multiple test cases to confirm functionality.
## A reference to a related issue in your repository (if applicable)
None 
## Checklist
- [X ] I can confirm that my changes are working as intended

AmmarMahzan committed:
I have completed Assignment 3

AmmarMahzan commented:
I have completed the Assignment 3.

AmmarMahzan committed:
Update assignment_3.md

Shiraz-Latif reviewed (changes requested):
Thanks for your submission! Please revisit the assignment instructions to ensure all requirements are met, it was required to build viz using two distinct tools for visualization. Let me know if you need support reworking or resubmitting!

AmmarMahzan committed:
Add files via upload

I have resubmitted Assignment-3 which includes all the files (├── assignment_3.md
├── Assigment_3.ipynb
├── viz1_description.md
├── viz2_description.md
├── Ticket Sales and Redemption Over Time.png
├── Average Sales and Redemptions by Hour.png
├── Toronto Island Ferry Ticket Counts.csv and Toronto Island Ferry Ticket Counts.json

AmmarMahzan committed:
Delete 02_activities/assignments/avg_sales_redemption_by_hour.png

AmmarMahzan committed:
Delete 02_activities/assignments/Toronto Island Ferry Ticket Counts.json

AmmarMahzan committed:
Delete 02_activities/assignments/Toronto Island Ferry Ticket Counts.csv

AmmarMahzan committed:
Delete 02_activities/assignments/Ticket Sales and Redemptions Over Time.png

AmmarMahzan committed:
Delete 02_activities/assignments/Assigment_3.ipynb

AmmarMahzan committed:
Update assignment_3.md

Shiraz-Latif reviewed (approved):
Thanks for the update - marked as complete
